packages/styles/src/table/table-cell.css

```diff
@@ -0,0 +1,128 @@
+/**
+ * Table cell classes
+ *
+ * @boolean { true } syn-table-cell Applies the table cell styling
+ * @boolean syn-table-cell--bg-neutral-50 Applies the table cell styling with a neutral background color
```

Comment:
I think `syn-table-cell--bg-neutral-50` and `syn-table-cell--bg-neutral-0` are quite long. Do we need names that are longer to type than to apply with our own css?

```css
.syn-table-cell--bg-neutral-50 {
    background-color: var(--syn-color-neutral-50);
}
```

vs.

```css
/* TODO: Find naming */
.syn-table-cell--light-grey {
    background-color: var(--syn-color-neutral-50);
}
```